worker/worker-manager/score-worker-manager-impl/src/main/java/io/cloudslang/worker/management/services/InBuffer.java

```diff
@@ -41,38 +41,41 @@
     private static final Logger logger = Logger.getLogger(InBuffer.class);
 
     private final static long MEMORY_THRESHOLD = 50000000; // 50 Mega byte
```

Comment:
This is the original inBuffer code, nothing new here, and i don't this particular parameter should be left for the user discretion